packages/block-library/src/image/edit.js

```diff
@@ -190,6 +190,7 @@ class ImageEdit extends Component {
 			...pickRelevantMediaFiles( media ),
 			width: undefined,
 			height: undefined,
+			sizeSlug: DEFAULT_SIZE_SLUG,
```

Comment:
Is `large` size enforced to exist for all media files?

Reply:
Looking at `pickRelevantMediaFiles()`, it looks like we already assumed that a `'large'` size exists.

Reply:
yes.